app/components/IndividualCampus.js

```diff
@@ -0,0 +1,44 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import store from '../store.jsx'
+import Navbar from './Navbar'
+import {getIndividualCampusThunkCreator} from '../reducers/index.jsx'
+
+
+class listCampusStudents extends Component{
+  constructor(){
+    super();
+  }
+
+  componentDidMount(){
+    const getIndividualCampusThunk = getIndividualCampusThunkCreator(this.props.match.params.campusId);
```

Comment:
I'm a fan of these long, verbose names.

A lot of programmers like to save characters when they are writing things out and create terse names.

But reading/understanding old code tends to be the more critical job. The clarity between the thunk creator and the thunk itself is helpful here.